site/content/en/docs/tasks/run/kubeflow/pytorchjobs.md

```diff
@@ -52,3 +52,33 @@ By default, Kueue will set `suspend` to true via webhook and unsuspend it when t
 This example is based on https://github.com/kubeflow/training-operator/blob/855e0960668b34992ba4e1fd5914a08a3362cfb1/examples/pytorch/simple.yaml.
 
 {{< include "examples/jobs/sample-pytorchjob.yaml" "yaml" >}}
+
+
+## MultiKueue integration
```

Comment:
I would suggest moving the content to a dedicated page under:
`Run Workloads / In MultiKueue / Training Operator`

Opened: https://github.com/kubernetes-sigs/kueue/issues/3134